src/components/event-listing/EventListing.tsx

```diff
@@ -48,13 +49,10 @@ const EventListing: React.FC<EventProps> = ({
   title,
   details,
 }) => {
-  const defaultImg =
-    'https://today.tamu.edu/wp-content/uploads/2019/03/GettyImages-184621282.jpg';
-
   return (
     <StyledCard>
       <CardContent>
-        <Thumbnail src={thumbnail || defaultImg} />
+        <Thumbnail src={thumbnail || DEFAULT_IMAGE} />
```

Comment:
Another related thought: could you make `DEFAULT_IMAGE` a default function parameter? 
`thumbnail = DEFAULT_IMAGE` in function params? Not sure if that's proper syntax.

Reply:
I tried to do this in the prop deconstruction but it didnt work :/ I'm gonna gonna do it with ||